internal/envoy/v3/listener.go

```diff
@@ -296,7 +309,7 @@ func (b *httpConnectionManagerBuilder) DefaultFilters() *httpConnectionManagerBu
 	// identified by the TypeURL of each filter.
 	b.filters = append(b.filters,
 		&http.HttpFilter{
-			Name: "compressor",
+			Name: CompressorFilterName,
```

Comment:
I _think_ it should be fine to change the filter names here as they're added to the HCM, but an alternate option would be to only update the per-filter config within the RouteConfigurations to match these names, without changing them here. Any thoughts?